TUTORIAL.md

```diff
@@ -434,6 +436,20 @@ end
 
 For an argument, each function will be tried in order. The first function that doesn't raise a `ContractError` will be used. So in this case, if x == 1, the first function will be used. For all other values, the second function will be used.
 
+This allows you write methods more declaratively, rather than using conditionals. This feature is not only useful for recursion; you can use it to keep parallel use cases separate:
+
+```ruby
+Contract lambda{|n| n < 12 } => Ticket
+def get_ticket(age)
+  ChildTicket.new(age: age)
+end
+
+Contract lambda{|n| n >= 12 } => Ticket
```

Comment:
I think we can use lambda in case 2, but also show `Num` as alternative, so they have a concrete example of overloads being checked in order of declaration. A simple case like this probably doesn't need a lambda for case 2, but we can mention both with a brief explanation of the benefits of each style.